src/main/java/org/apache/sling/resourceresolver/impl/mapping/MapEntries.java

```diff
@@ -341,7 +342,8 @@ private void execute() {
                 // drain once more in case more events have arrived
                 drainQueue(resourceChangeQueue);
 
-                log.debug("vanity path init - end");
+                long initElapsed = System.nanoTime() - initStart;
+                log.info("vanity path initialization - end, elapsed {}ms", TimeUnit.NANOSECONDS.toMillis(initElapsed));
```

Comment:
I think we should have some consistency in the log message format. Below when the exception is logged it is "VanityPath", here (and above) it is "vanity path"; I haven't checked the whole code - but at least here using "VanityPath" seems to be more consistent